frontend/src/components/SelectItems.vue

```diff
@@ -106,14 +106,21 @@ export default {
   name: "SelectItems",
   mounted() {
     var userId = firebase.auth().currentUser.uid;
-    api
-      .getAllItemsInCart(userId)
-      .then(response => {
-        this.itemsInCart = response.data;
-      })
-      .catch(error => {
-        this.$log.debug(error);
-        this.error = "Failed to load data";
+    var tempArray = [];
+    let self = this;
+    firebase
+      .database()
+      .ref("user_cart/" + userId)
+      .once("value")
+      .then(function(snapshot) {
```

Comment:
**snapshot** can be renamed as the **cart**
Similarly **childSnapshot** can be renamed as the **cartItem**